### PR TITLE
Improve 401 handling

### PR DESCRIPTION
**Pull request by zoontek**
- Don't show an error when user is logged out as the error exchange will redirect to the login page
- Don't use suspense for auth status check, perform project infos + auth status in parallel (avoid blinking loader issue)
- Uniformize loading view color: grey 400 + background color outside of project scope, current 500 inside

### CODE REVIEW

File: clients/banking/src/App.tsx [hunk 1]
```diff
@@ -34,20 +34,9 @@ export const App = () => {
...
-              )
-            : false;
-        return is401 ? (
-          <LoadingView color={colors.gray[400]} style={styles.base} />
-        ) : (
-          <ErrorView error={error} style={styles.base} />
-        );
-      }}
+      fallback={({ error }) =>
+        isUnauthenticatedError(error) ? <></> : <ErrorView error={error} style={styles.base} />
```
Comment (Epimodev) [inline clients/banking/src/App.tsx:38]: nit: will it be similar to return `null` instead of `<></>`?

**Review by Epimodev — APPROVED** [review]
just 1 question but this is good for me. Thanks for taking care about those edge cases not easy to reproduce

File: clients/banking/src/App.tsx (hunk 1 above)
Comment (zoontek) [reply clients/banking/src/App.tsx:38]: You can't, the typing expect a `ReactElement`

Screenshot 2023-05-30 at 15 16 05

Comment (Epimodev) [reply clients/banking/src/App.tsx:38]: perfect thanks a lot!

**Pull request merged**